### DISCUSSION TITLE
valve ne se ferme pas toujours

### DISCUSSION THREAD

**Discussion by Lux-WorldPC** [jmcollin78/versatile_thermostat#530] [General]
bonjour,
ce n'est pas systématique, mais souvent les valves ne se ferment pas 100%
ci dessous un example, je demande 20.4 il mesure 22.9 et les valves sont toujours a 5%
j'ai fait une erreur ?
merci pour votre aide
François

hvac_modes:
  - heat
  - "off"
min_temp: 7
max_temp: 35
target_temp_step: 0.1
preset_modes:
  - none
  - frost
  - eco
  - comfort
  - boost
current_temperature: 22.9
temperature: 20.4
hvac_action: heating
preset_mode: eco
is_on: true
hvac_mode: heat
type: null
is_controlled_by_central_mode: true
last_central_mode: Auto
frost_temp: 10
eco_temp: 20.4
boost_temp: 25
comfort_temp: 22.4
frost_away_temp: 18
eco_away_temp: 18
boost_away_temp: 18
comfort_away_temp: 18
power_temp: null
target_temperature_step: 0.1
ext_current_temperature: 8.4
ac_mode: false
current_power: null
current_power_max: null
saved_preset_mode: eco
saved_target_temp: 10
saved_hvac_mode: null
motion_sensor_entity_id: null
motion_state: null
power_sensor_entity_id: null
max_power_sensor_entity_id: null
overpowering_state: null
presence_sensor_entity_id: null
presence_state: null
window_state: "off"
window_auto_state: "off"
window_bypass_state: false
window_sensor_entity_id: null
window_delay_sec: null
window_auto_enabled: false
window_auto_open_threshold: null
window_auto_close_threshold: null
window_auto_max_duration: null
window_action: window_turn_off
security_delay_min: 60
security_min_on_percent: 0.5
security_default_on_percent: 0.1
last_temperature_datetime: "2024-10-08T23:05:12.814221+02:00"
last_ext_temperature_datetime: "2024-10-08T23:00:00.006402+02:00"
security_state: false
minimal_activation_delay_sec: 300
device_power: 1
mean_cycle_power: 0
total_energy: 0
last_update_datetime: "2024-10-08T23:09:02.449869+02:00"
timezone: Europe/Paris
temperature_unit: °C
is_device_active: true
ema_temp: 22.81
is_used_by_central_boiler: false
valve_open_percent: 5
is_over_valve: true
underlying_valve_0: number.rdc_cuis_rad_valve_position
underlying_valve_1: number.rdc_hal_rad_valve_position
underlying_valve_2: number.rdc_sal_rad_valve_position
underlying_valve_3: number.rdc_sam_rad_valve_position
on_percent: 0
on_time_sec: 0
off_time_sec: 600
cycle_min: 10
function: tpi
tpi_coef_int: 0.7
tpi_coef_ext: 0.01
auto_regulation_dpercent: 5
auto_regulation_period_min: 10
last_calculation_timestamp: "2024-10-08T21:53:45.801004+02:00"
friendly_name: Therm_Entree
supported_features: 401

**Comment by Lux-WorldPC** [comment]
toutes mes excuses, je viens de lire ça
je viens de remettre tpi_coef_int: 0.6
je voir jusque demain comment ça se comporte.
merci

The radiator heats up even though the setpoint temperature is exceeded or does not heat up even though the room temperature is well below the setpoint
Type over_switch or over_valve
With a VTherm of type over_switch or over_valve, this fault just shows that the parameters of the TPI algorithm are incorrectly set. See [TPI Algorithm](https://ha1.lwdns.lu:8123/hacs#tpi-algorithm) to optimize the settings.

**Comment by Lux-WorldPC** [comment]
<img width="717" alt="Screenshot 2024-10-09 at 16 58 10" src="https://github.com/user-attachments/assets/121f0083-9d7d-4246-8147-5d810237ef7a">
la j'ai TPI par défaut     et reste encore bloqué a 3%.      un simple restart HA     corrige le problème tout seul.

**Comment by jmcollin78** [comment]
Hello, thank you for your report. It is certainly due to auto_regulation_dpercent: 5.
I will fix that rapidly. Wiating for the fix you can: set auto_regulation_dpercent to 0 and keep only auto_regulation_period_min: 10. The behaviour should be quasi the same

**Reply by Lux-WorldPC** [reply]
super merci beaucoup pour l'info.    many thanks

**Reply by Lux-WorldPC** [reply]
update installed and HA rebooted

**Reply by jmcollin78** [reply]
Keep me in touch with the new release but someone already says it was ok.

**Comment by jmcollin78** [comment]
Salut @Lux-WorldPC ,

Quel matériel utilises-tu comme vanne ? Marque et modèle stp.
